src/components/feature/FeatureCard.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+
+import { FeatureStatus, IFeature } from "@types";
+import { constants, readableDate, readableTime } from "@utils";
+import { Separator } from "@components/Separator";
+
+interface IFeatureCardProps {
+	feature: IFeature
+}
+
+const Timestamp = ({ date }: { date: Date }) => {
+	return (
+		<div className="text-[11px] italic text-theme-gray">
+			<div className="inline-flex pr-1 font-bold align-middle">
+				{
+					readableDate(new Date(date).toISOString())
+				}
+			</div>
+			:
+			<div className="inline-flex pl-1 font-bold align-middle">
+				{
+					readableTime(new Date(date).toISOString())
+				}
+			</div>
+		</div>
+	);
+};
+
+const PosterInfo = ({ feature }: { feature: IFeature }) => {
+	return (
+		<div className="inline-flex pr-1 text-theme-gray">
+			Suggested By - {feature.user.username}
+		</div>
+	);
+};
+
+const statusColorLookup: Record<FeatureStatus, string> = { 
+	[FeatureStatus.pending] : "theme-yellow",
+	[FeatureStatus.completed] : "theme-green", 
+	[FeatureStatus.rejected] : "theme-red",
+	[FeatureStatus.in_progress]: "theme-yellow"
+};
+
+const PendingStatus = ({ feature }: { feature: IFeature }) => {
+	return (
+		<div className="inline-flex pt-1 pl-0.5 text-[11px] align-middle">
+			<div className={`mr-1 text-${statusColorLookup[feature.status]}`}># {feature.status}</div>
+			<div className="inline-flex pr-1 font-bold align-middle text-theme-gray">
+				at
+				<div className="mx-1">
```

Comment:
you already have a readable date component, dont make another one here, just make it reusable